src/css/base.css

```diff
@@ -27,3 +58,9 @@ body {
 main {
   flex-grow: 1;
 }
+
+.container {
+  margin: 0 auto;
+  padding: 0 20px;
+  max-width: 375px;
+}
```

Comment:
we already have styles for container at the container.css, please remove it and update there if needed